guides/keras_cv/semantic_segmentation_deeplab_v3_plus.py

```diff
@@ -0,0 +1,298 @@
+"""
+Title: Image Semantic Segmentation with DeepLabV3Plus
+Author: Ian Stenbit, Divyashree Sreepathihalli
```

Comment:
You should be first author -- you did all the writing!

Reply:
Thanks Ian! Updated!